crates/bitcoind_rpc/tests/test_emitter.rs

```diff
@@ -166,17 +20,22 @@ impl TestEnv {
 #[test]
 pub fn test_sync_local_chain() -> anyhow::Result<()> {
     let env = TestEnv::new()?;
-    let (mut local_chain, _) = LocalChain::from_genesis_hash(env.client.get_block_hash(0)?);
-    let mut emitter = Emitter::new(&env.client, local_chain.tip(), 0);
+    let tip = env.rpc_client().get_block_count()?;
+    let (mut local_chain, _) = LocalChain::from_genesis_hash(env.rpc_client().get_block_hash(0)?);
+    let mut emitter = Emitter::new(env.rpc_client(), local_chain.tip(), 0);
 
-    // mine some blocks and returned the actual block hashes
+    // Mine some blocks and return the actual block hashes.
+    // We must first add the hashes of the genesis block and the block automatically generated from
+    // initializing `ElectrsD`.
```

Comment:
Nit: I think this is clearer?

```suggestion
    // Mine some blocks and return the actual block hashes.
    // Because initializing `ElectrsD` already mines some blocks, we must include those too when returning block hashes.
```